backend/src/data/repositories/instance/instance.repository.ts

```diff
@@ -58,17 +61,16 @@ class Instance {
   }: {
     filter: SCInstanceGetByTenantRequestParamsDto;
     tenantId: string;
-  }): Promise<InstanceEntity[]> {
+  }): Promise<InstanceM[]> {
```

Comment:
```suggestion
  }): Promise<InstanceEntity[]> {
```
You can add operationSystem property yo your entity